core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -188,6 +190,7 @@ public static void unset(PropertyKey key) {
    * @param key the key to get the value for
    * @return the value for the given key
    */
+  @SuppressWarnings("nullness")
```

Comment:
I added this ```@SuppressWarnings("nullness")``` in the first place because the function ```lookup(rawValue)``` it's null when the function ```lookup``` it's called with a ```null``` string. This is a false positive report from the Checker Framework since this code will throw an exception in those cases.

I replaced the warning suppression with an assertion:
```
String value = lookup(rawValue);
assert value != null : "@AssumeAssertion(nullness)";
return value;
```